### PR TITLE
Added RSS feed examples and minor clarifications

### PR DESCRIPTION
**Pull request by jonsnielsen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,11 +34,12 @@ To build the zeeguu-api-core container image:
 docker build -t zeeguu-api-core -f docker-files/zeeguu-api-core/Dockerfile .
 ```
 
-To build the zeeguu-web container image:
+To build the zeeguu-web container image:   
+(Make sure to replace 1.1.1.1:9001 with the url:port where your API can be reached from other clients.)
```
Comment (alinbalutoiu) [inline README.md]: I think this fits better on the previous line, for example:
```
To build the zeeguu-web container image (make sure to replace 1.1.1.1:9001 with the url:port where your API can be reached from other clients):
```

File: README.md [hunk 2]
```diff
@@ -34,11 +34,12 @@ To build the zeeguu-api-core container image:
...
 ```sh
 docker build -t zeeguu-web --build-arg ZEEGUU_API__EXTERNAL="http://1.1.1.1:9001"  -f docker-files/zeeguu-web/Dockerfile .
 ```
-Make sure to replace ``1.1.1.1:9001`` with the url:port where your API can be reached from other clients.
+<!-- Make sure to replace ``1.1.1.1:9001`` with the url:port where your API can be reached from other clients. -->
```
Comment (alinbalutoiu) [inline README.md]: This should be removed instead of commented out.

**Review by alinbalutoiu — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jonsnielsen) [reply README.md]: Ups, the comment was a mistake. made the changes

File: README.md [hunk 3]
```diff
@@ -113,3 +113,12 @@ To fetch the articles from the RSS feed, run the following:
 ```sh
 docker exec -i zeeguu-api-core python /opt/Zeeguu-Core/tools/feed_retrieval.py
 ```
+
+Some RSS feed examples to get started:
```
Comment (alinbalutoiu) [inline README.md]: Can you also make this a sub-header? (i.e.: ``#### RSS feed examples``).
And then make the below list to be a bullet list (add ``* `` at the start of each line).

**Review by alinbalutoiu — COMMENTED** [review]
Just one more small comment, missed it last time. After that just squash all the commits and make a single one with all the changes and it's good to go.

**Comment by jonsnielsen** [comment]
Changes done. As far as I understand only the ones with write acess can squash the commits? 
Found it here https://help.github.com/articles/about-pull-request-merges/

**Comment by alinbalutoiu** [comment]
You are sending the pull request from your branch. One way would be to do a ``git reset`` to remove all the commits and then ``git commit`` to add one single commit with all the changes. After that a ``git push --force`` should do the job. You might want to use ``git commit --amend`` in future which adds your changes to the latest commit instead of creating a new commit.

**Comment by jonsnielsen** [comment]
My mistake closed the pull request earlier.
Things should be good now, only one commit made :) 
Thanks for the git advices Alin!

**Review by alinbalutoiu — APPROVED** [review]
Looks good!

**Pull request merged**